mappings/net/minecraft/client/render/block/BlockRenderManager.mapping

```diff
@@ -1,13 +1,31 @@
 CLASS net/minecraft/class_776 net/minecraft/client/render/block/BlockRenderManager
+	FIELD field_20987 blockColors Lnet/minecraft/class_324;
 	FIELD field_4167 fluidRenderer Lnet/minecraft/class_775;
 	FIELD field_4168 models Lnet/minecraft/class_773;
 	FIELD field_4169 random Ljava/util/Random;
-	FIELD field_4170 renderer Lnet/minecraft/class_778;
-	METHOD method_23071 tesselateDamage (Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;Lnet/minecraft/class_1920;Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;)V
+	FIELD field_4170 blockModelRenderer Lnet/minecraft/class_778;
+	METHOD method_23071 renderDamage (Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;Lnet/minecraft/class_1920;Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;)V
+		ARG 1 state
+		ARG 2 pos
+		ARG 3 world
+		ARG 4 matrix
+		ARG 5 vertexConsumer
 	METHOD method_3349 getModel (Lnet/minecraft/class_2680;)Lnet/minecraft/class_1087;
 		ARG 1 state
 	METHOD method_3350 getModelRenderer ()Lnet/minecraft/class_778;
 	METHOD method_3351 getModels ()Lnet/minecraft/class_773;
-	METHOD method_3352 tesselateFluid (Lnet/minecraft/class_2338;Lnet/minecraft/class_1920;Lnet/minecraft/class_4588;Lnet/minecraft/class_3610;)Z
-	METHOD method_3353 renderOnEntity (Lnet/minecraft/class_2680;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;II)V
-	METHOD method_3355 tesselateBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;Lnet/minecraft/class_1920;Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;ZLjava/util/Random;)Z
+	METHOD method_3352 renderFluid (Lnet/minecraft/class_2338;Lnet/minecraft/class_1920;Lnet/minecraft/class_4588;Lnet/minecraft/class_3610;)Z
+	METHOD method_3353 renderBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;II)V
```

Comment:
This one is different in that it uses different render layers than those of regular blocks. Beware!